### DISCUSSION TITLE
Double-entry bookkeeping

### DISCUSSION THREAD

**Discussion by VictorVow** [ghostfolio/ghostfolio#3428] [Q&A]
I couldn't find an answer for this - is this feature present already? If not, then it is planned?

**Comment by dtslvr** [comment]
Hi @Victor239

Ghostfolio currently does not support double-entry bookkeeping. Could you explain why you need this feature and how you would use it? Understanding your needs could help us consider it for future updates.

**Reply by VictorVow** [reply]
I use it currently with the personal finance app GnuCash, I like it a lot as it helps ensure that transaction amounts are accurately tracked, so before moving to this app I'd like to see it here too.
